crrExistingObjects.js

```diff
@@ -223,13 +243,43 @@ function _listObjectVersions(bucket, VersionIdMarker, KeyMarker, cb) {
         Prefix: TARGET_PREFIX,
         VersionIdMarker,
         KeyMarker,
-    }, cb);
+    }, (err, data) => {
+        if (err) {
+            // retry only on Network errors and if the process is set to retry
+            // on errors.
+            if (err.code === 'NetworkingError' && RETRY_ON_ERROR) {
+                log.error('Error listing object versions, retrying request',
+                    { error: err, bucketName: bucket });
+                return _listObjectVersions(bucket, VersionIdMarker,
+                    KeyMarker, cb);
+            }
+            return cb(err);
```

Comment:
Log at error level here

Reply:
Yes, the callback is logging it as error.